### PR TITLE
[Enhancement] Keyboard shortcut for running queries

### PR DESCRIPTION
**Pull request by sumukhswamy**
### Description

cmd+enter functionality runs the query in the editor

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/data/public/ui/query_editor/query_editor_top_row.tsx [hunk 1]
```diff
@@ -72,6 +72,20 @@ export default function QueryEditorTopRow(props: QueryEditorTopRowProps) {
     },
   } = opensearchDashboards.services;
 
+  useEffect(() => {
+    function handleCmdEnter(event: KeyboardEvent) {
```
Comment (kavilla) [inline src/plugins/data/public/ui/query_editor/query_editor_top_row.tsx:76]: I think we could use 

this.inputRef which is the Monaco Editor ref and add an action like [this](https://microsoft.github.io/monaco-editor/playground.html?source=v0.52.0#example-interacting-with-the-editor-adding-an-action-to-an-editor-instance)

Comment (sumukhswamy) [reply src/plugins/data/public/ui/query_editor/query_editor_top_row.tsx]: Have updated it

File: src/plugins/data/public/ui/query_editor/query_editor.tsx [hunk 2]
```diff
@@ -366,6 +366,20 @@ export default class QueryEditorUI extends Component<Props, State> {
...
+        const handleEnterPress = () => {
+          this.onSubmit(this.props.query);
+        };
+
+        const disposable = editor.onKeyDown((e) => {
+          if (e.metaKey && e.keyCode === monaco.KeyCode.Enter) {
+            e.preventDefault();
+            handleEnterPress();
+          }
+        });
```
Comment (ashwin-pc) [inline src/plugins/data/public/ui/query_editor/query_editor.tsx:378]:
```suggestion
        const disposable = editor.onKeyDown((e) => {
          if (e.metaKey && e.keyCode === monaco.KeyCode.Enter) {
            e.preventDefault();
            this.onSubmit(this.props.query);
          }
        });
```
Nit: handleEnterPress doesnt seem to add any value here. What do you think?

**Comment by ashwin-pc** [comment]
@sumukhswamy can we add a changelog file here?

**Review by ashwin-pc — APPROVED** [review]

**Review by sejli — APPROVED** [review]
Rerunning failed test, should just a random failure from not starting up correctly. Otherwise, LGTM!

**Pull request merged**